examples/gke/xfs-formatter-daemonset.yaml

```diff
@@ -0,0 +1,95 @@
+# Daemonset that converts ephemeral-storage to xfs
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: xfs-formatter
+spec:
+  selector:
+    matchLabels:
+      app: xfs-formatter
+  template:
+    metadata:
+      labels:
+        app: xfs-formatter
+    spec:
+      hostPID: true
+      nodeSelector:
+        cloud.google.com/gke-ephemeral-storage-local-ssd: "true"
```

Comment:
You'll need to elaborate on why is this needed given the admin has already labeled nodes with the other one.

Reply:
I've added it after Valerii tried to deploy the DS without the beta feature enabled, so that it would be somewhat more robust, but I suppose this label could be added to a node anyway, so maybe this is redundant with the other label. Wdyt?

Reply:
@tnozicka My proposed solution would be to remove the `scylla.scylladb.com/gke-ephemeral-storage-local-ssd=true` label and only leave the one created by GKE, i.e. `cloud.google.com/gke-ephemeral-storage-local-ssd: "true"`. Instead of the additional label, we could add the following toleration to the DaemonSet:
```
  tolerations:
    - key: role
      operator: Equal
      value: scylla-clusters
      effect: NoSchedule
```
which is probably a more robust way to approach it.

Reply:
I think it's correct to migrate over to the GKE label.

Not sure what you aim for with the toleration instead of node selector. The selector restricts "which nodes to land on" while the toleration means "allow landing on these node" (but also elsewhere).

Reply:
Right, that didn't make much sense. I've changed the set of nodeSelectors to two GKE provided ones:
```
      nodeSelector:
        cloud.google.com/gke-ephemeral-storage-local-ssd: "true"
        cloud.google.com/gke-nodepool: "scylla-pool"
```